rotkehlchen/globaldb/handler.py

```diff
@@ -813,12 +813,11 @@ def get_ethereum_tokens(
                 querystr_additions.append(f'(B.protocol NOT IN ({",".join(questionmarks)}) OR B.protocol IS NULL) ')  # noqa: E501
                 bindings_list.extend(except_protocols)
 
-            querystr += 'WHERE ' + 'AND '.join(querystr_additions) + ';'
-            bindings = tuple(bindings_list)
+            querystr += 'AND ' + 'AND '.join(querystr_additions) + ';'
```

Comment:
Hmmm if querystr_additions are empty what happens here? Does it not end with an `AND;`?

Reply:
If the code enters this if statement is because it holds that `exceptions is not None or protocol is not None or except_protocols is not None`. The only way for it to not have the `querystr_additions` is that `len(exceptions)==0` or `len(except_protocols)==0` what I would say is a logic error. As it was before it would also add the extra `WHERE` and fail but I can change it to be protected against this kind of error

Reply:
Ah true yes. I mean lol that was also the case before.